data.sql

```diff
@@ -3,4 +3,11 @@
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  overriding system value values (1, 'Agumon', '03-02-2020',0, true, 10.23);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  overriding system value values (2, 'Gabumon', '2018-11-15',2, true, 8.00);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  overriding system value values (3, 'Pikachu', '07-01-2021',1, false, 15.04);
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  overriding system value values (4, 'Agumon', '12-05-2017',5, true, 10.23);
\ No newline at end of file
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  overriding system value values (4, 'Devimon', '12-05-2017',5, true, 10.23);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  overriding system value values (5, 'Charmander', '08-02-2020',0, false, -11.00);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  overriding system value values (6, 'Plantmon', '15-12-2021',2, true, -5.7);
```

Comment:
- [ ] Date of birth of Plantom should be `'2022/11/15'`, this is important because on the next milestone you will have to run queries, and because of incorrect inserted date of birth you may get less data in response